[BUG] panic on `lotus client query-ask`

**Describe the bug**
For a subset of miner id's, when I try to run `lotus client query-ask <miner id>`, I get a panic due to nil pointer dereference.  Example:

```
$ lotus client query-ask f0680676
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1649fcc]

goroutine 1 [running]:
github.com/filecoin-project/lotus/cli.glob..func22(0xc0003513c0, 0x0, 0x0)
	/home/ubuntu/lotus-jdcloud/cli/client.go:1559 +0xaec
github.com/urfave/cli/v2.(*Command).Run(0x50ff940, 0xc000351240, 0x0, 0x0)
	/home/ubuntu/go/pkg/mod/github.com/urfave/cli/v2@v2.2.0/command.go:164 +0x4dd
github.com/urfave/cli/v2.(*App).RunAsSubcommand(0xc000882a80, 0xc000350f40, 0x0, 0x0)
	/home/ubuntu/go/pkg/mod/github.com/urfave/cli/v2@v2.2.0/app.go:427 +0xa49
github.com/urfave/cli/v2.(*Command).startApp(0x51005a0, 0xc000350f40, 0x7fff7e8296f1, 0x6)
	/home/ubuntu/go/pkg/mod/github.com/urfave/cli/v2@v2.2.0/command.go:279 +0x6bb
github.com/urfave/cli/v2.(*Command).Run(0x51005a0, 0xc000350f40, 0x0, 0x0)
	/home/ubuntu/go/pkg/mod/github.com/urfave/cli/v2@v2.2.0/command.go:94 +0x9cd
github.com/urfave/cli/v2.(*App).RunContext(0xc000882900, 0x3db7a70, 0xc000052090, 0xc00004e080, 0x4, 0x4, 0x0, 0x0)
	/home/ubuntu/go/pkg/mod/github.com/urfave/cli/v2@v2.2.0/app.go:306 +0x810
github.com/urfave/cli/v2.(*App).Run(...)
	/home/ubuntu/go/pkg/mod/github.com/urfave/cli/v2@v2.2.0/app.go:215
github.com/filecoin-project/lotus/cli.RunApp(0xc000882900)
	/home/ubuntu/lotus-jdcloud/cli/helper.go:35 +0x7c
main.main()
	/home/ubuntu/lotus-jdcloud/cmd/lotus/main.go:89 +0x6ff
```

**Version (run `lotus version`):**
```
$ lotus version
Daemon:  1.11.0-dev+mainnet+git.bf468762b.dirty+api1.3.0
Local: lotus version 1.11.0-dev+mainnet+git.bf468762b.dirty
```

**To Reproduce**
See above.

Here are two other miner id's that also produce the same error:  `f0224728`, `f0232830`

**Expected behavior**
Don't expect panic.  Expect output like this:

```
Ask: f08403
Price per GiB: 0.00000001 FIL
Verified Price per GiB: 0 FIL
Max Piece size: 32 GiB
Min Piece size: 128 MiB
```

**Logs**
Does the problem happen for you also?  I can't find the pattern `f0680676` anywhere in my logs.

--- a/cli/client.go
+++ b/cli/client.go
@@ -1759,7 +1759,7 @@ var clientQueryAskCmd = &cli.Command{
 				return xerrors.Errorf("failed to get peerID for miner: %w", err)
 			}
 
-			if *mi.PeerId == peer.ID("SETME") {
+			if mi.PeerId == nil || *mi.PeerId == peer.ID("SETME") {
 				return fmt.Errorf("the miner hasn't initialized yet")
 			}
 

